File: orderservice/Models/Product.cs
using System.ComponentModel.DataAnnotations;

public class Product{

    [Key]
    public Guid ProductGuid {get; set;}
    
    [Required]
    public string productName {get;set;}

    [Required]
    public string description {get;set;}

    [Required]
    public string rating {get;set;}

    [Required]
    public int price {get;set;}

    [Required]
    public Order Order {get;set;}

    [Required] 
    public Guid OrderGuid {get;set;}
}

File: basketservice/src/main/java/basketservice/basketservice/Product.java
package basketservice.basketservice;

import java.util.*;
import org.springframework.data.annotation.*;
import java.io.Serializable;


public class Product implements Serializable {

    private static final long serialVersionUID = 1L;

    @Id
    private UUID productGuid;
    private String productName;
    private String description;
    private String rating;
    private int price;

    public UUID getProductGuid() {
        return productGuid;
    }
    public void setProductGuid(UUID productGuid) {
        this.productGuid = productGuid;
    }
    public String getProductName() {
        return productName;
    }
    public void setProductName(String productName) {
        this.productName = productName;
    }
    public String getDescription() {
        return description;
    }
    public void setDescription(String description) {
        this.description = description;
    }
    public String getRating() {
        return rating;
    }
    public void setRating(String rating) {
        this.rating = rating;
    }
    public int getPrice() {
        return price;
    }
    public void setPrice(int price) {
        this.price = price;
    }
}
